mycelo/config/genesis_config.go

```diff
@@ -0,0 +1,454 @@
+package config
+
+import (
+	"fmt"
+	"math/big"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/consensus/istanbul"
+	"github.com/ethereum/go-ethereum/consensus/istanbul/uptime"
+	"github.com/ethereum/go-ethereum/mycelo/fixed"
+	"github.com/ethereum/go-ethereum/params"
+)
+
+// durations in seconds
+const (
+	Second = 1
+	Minute = 60 * Second
+	Hour   = 60 * Minute
+	Day    = 24 * Hour
+	Week   = 7 * Day
+	Year   = 365 * Day
+)
+
+type GenesisConfig struct {
```

Comment:
it seems that now, to generate the genesis we only need this struct and the genesis accounts, not env.json